### ISSUE TITLE
Don't schedule an immediate model update for quick fixes that wouldn't impact other files

### ISSUE THREAD

**Issue by davidfestal** [ceylon/ceylon-ide-intellij#464]
In commit ceylon/ceylon-ide-intellij@1945e0a, I added

`resumeAutomaticModelUpdate(0);` in various places.

In the cases that __*don't need*__ scheduling an immediate model update it should be:

`resumeAutomaticModelUpdate();`

(without the explicit `0` delay).

Cases that __*don't need*__ scheduling an immediate model updates are quick fixes whose change is not expected to impact any other file (to produce or fix an error for example). A typical example of such quick fixes is the `convert to named arguments` or the `convert to shortcut refinement`

**Comment by bjansen** [comment]
I added a new parameter `Boolean affectsOtherUnits = false` to `QuickFixDate.addQuickFix()`, so now each quick fix can decide if it affects other units or not. As a result, the plugin will know if it should run a model update immediately or after 4s.

I changed a few quick fixes with `affectsOtherUnits = true;`, but I might have missed some. @gavinking if you have a few minutes and want to review the usages of `addQuickFix`, that would be very helpful.
